README.md

```diff
@@ -12,6 +12,16 @@ _docs:_
 
 _Clojure version compatibility:_ 1.8 and up
 
+## Stewards
+
+matcher-combinators is maintained by:
+* Devflow team at Nubank
+* [Phillip Mates](https://github.com/philomates)
+
+For questions and more info:
+* Nubankers, please reach us on [#devflow](https://nubank.slack.com/archives/CS9MK8MGX)
+* Clojure community, please use the Clojurians channel [#matcher-combinators](https://clojurians.slack.com/archives/C04ABMF89D3)
```

Comment:
Maybe reorder these 2 lines?

Reply:
Also, I worry about conversations being bifurcated between Nubank and Clojurians Slack. WDYT? Just reference the public venue (Clojurians)?

Reply:
Yes, keeping the discussion in a single place might be better. I see no problem in just referencing Clojurians. WDYT, @acamargo ?
For the other OSS libraries we were considering either creating their own channels on Clojurians or enabling the "Discussions" space in their repositories. Any thoughts?

Reply:
@kdmoreira I'm OK having a single place for discussion, and that place being Clojurians.